scipy/stats/tests/test_kdeoth.py

```diff
@@ -316,24 +316,28 @@ def test_kde_integer_input():
     assert_array_almost_equal(kde(x1), y_expected, decimal=6)
 
 
-_ftypes = [getattr(np, dtype) for dtype in ['float32', 'float64', 'float96',
-                                            'float128', 'int32', 'int64']
-           if hasattr(np, dtype)]
+_ftypes = ['float32', 'float64', 'float96', 'float128', 'int32', 'int64']
```

Comment:
This change leads to an extra 175 tests being generated, but it works and is just as fast, so not a real issue (just commenting for people looking back at this later; no change needed).